content/install/installation_overview.md

```diff
@@ -82,7 +86,7 @@ services:
       - DRONE_SECRET=${DRONE_SECRET}
 ```
 
-Drone mounts a volume on the host machine to persist the sqlite database.
+Drone mounts a [data volume](https://docs.docker.com/storage/volumes/#create-and-manage-volumes) named "drone-server-data" to persist the sqlite database.
```

Comment:
This isn't right, the volume will be named foldername-drone-server-data because if docker-compose

Reply:
👍 It is right. Since it can depends a lot of the end user context (could be prefixed by the value of `$COMPOSE_PROJECT` or the value of the docker-compose's flag `-p `), I've removed this mention to avoid confusion. WDYT?